Tasks/AzureResourceManagerTemplateDeploymentV3/models/DeployAzureRG.ts

```diff
@@ -0,0 +1,94 @@
+import tl = require("azure-pipelines-task-lib/task");
```

Comment:
need to remove this file?

Reply:
Yes, came back as a part of merge I guess. Deleted.